src/Resources/contao/forms/FormFieldsetStop.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * Contao Open Source CMS
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao;
+
+
+/**
+ * Class FormFieldsetSTop
+ *
+ * @author Leo Feyer <https://github.com/leofeyer>
+ */
+class FormFieldsetStop extends \Widget
+{
+
+	/**
+	 * Template
+	 *
+	 * @var string
+	 */
+	protected $strTemplate = 'form_fieldsetStop';
+
+
+	/**
+	 * Do not validate
+	 */
+	public function validate()
+	{
+		return;
+	}
+
+
+	/**
+	 * Parse the template file and return it as string
+	 *
+	 * @param array $arrAttributes An optional attributes array
+	 *
+	 * @return string The template markup
+	 */
+	public function parse($arrAttributes=null)
+	{
+		// Return a wildcard in the back end
+		if (TL_MODE == 'BE')
+		{
+			/** @var BackendTemplate|object $objTemplate */
+			$objTemplate = new \BackendTemplate('be_wildcard');
+
+			return $objTemplate->parse();
+		}
+
+		return parent::parse($arrAttributes);
+	}
+
+
+	/**
+	 * Generate the widget and return it as string
+	 *
+	 * @return string The widget markup
+	 */
+	public function generate()
+	{
+		return "  </fieldset>\n";
```

Comment:
I wonder if we should throw an exception here? Ever since we've added the `Widget::parse()` method in Contao 3.3, using `Widget::generate()` was no longer recommended because it bypasses the template system. The existing widget classes still support the method to remain backwards compatibility, but this is a new class, so …

@contao/developers What do you think?